lib/auth0/api/authentication_endpoints.rb

```diff
@@ -532,18 +532,14 @@ def validate_id_token(id_token, algorithm: nil, leeway: 60, nonce: nil, max_age:
           raise Auth0::InvalidParameter, 'Must supply a valid max_age'
         end
 
-        unless algorithm.nil? || algorithm.is_a?(Auth0::Algorithm::RS256) || algorithm.is_a?(Auth0::Algorithm::HS256)
-          raise Auth0::InvalidParameter, "Signature algorithm of \"#{algorithm}\" is not supported"
-        end
-
         context = {
           issuer: issuer || "https://#{@domain}/",
           audience: audience || @client_id,
           algorithm: algorithm || Auth0::Algorithm::RS256.jwks_url("https://#{@domain}/.well-known/jwks.json"),
-          leeway: leeway
+          leeway: leeway,
```

Comment:
We agreed you will change this back to how it was before.

Reply:
Changed in 64bc3aa057870bb29267f94fd0333c74b958ed82